libbeat/scripts/Makefile

```diff
@@ -440,7 +440,7 @@ prepare-package-cgo:
 		-e BEAT_NAME=${BEAT_NAME} \
 		${BEATS_BUILDER_IMAGE}
 
-	# linux builds on debian 6 for compatibility
+	# linux builds on older debian for compatibility
```

Comment:
I remember we had debian 6 here because for some specific compatibility which was not the case with newer debian versions. Does this change mean we stop supporting Debian 6? And do we still need the old debian builds? @tsg might now more here.